src/common/components/dropdown/components/option-group.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import classes from '../dropdown.component.module.css';
+import { Tick } from '../../icons/tick-icon.component';
+import { DropdownOptionVm } from '../dropdown.model';
+
+interface Props {
+  name: string;
+  options: DropdownOptionVm[];
+  optionsListVisible: boolean;
+  handleOptionClick: (option: DropdownOptionVm) => void;
+  selectedPKField?: string;
```

Comment:
Since we have done this component generci, let's name it _selectedOption_

Reply:
It's fixed.